robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow: /0.5/*
```

Comment:
just `Disallow: /0.5/` to get the entire directory. http://www.robotstxt.org/robotstxt.html